scope_registry.go

```diff
@@ -263,17 +263,17 @@ func (r *scopeRegistry) reportInternalMetrics() {
 	histograms.Add(rootHistograms.Load())
```

Comment:
Let's also add a gauge metric for num-active-scopes

Reply:
done